qiskit/providers/aer/aerprovider.py

```diff
@@ -49,3 +50,9 @@ def backends(self, name=None, filters=None, **kwargs):
 
     def __str__(self):
         return 'AerProvider'
+
+    @staticmethod
+    def optimize_backend_options(min_qubits=10, max_qubits=20, ntrials=10):
```

Comment:
Lets remove this method from the provider for now and make it have to be run by importing the profiling functions